d2/src/main/java/com/linkedin/d2/xds/balancer/XdsLoadBalancerWithFacilitiesFactory.java

```diff
@@ -0,0 +1,37 @@
+package com.linkedin.d2.xds.balancer;
+
+import com.linkedin.d2.balancer.D2ClientConfig;
+import com.linkedin.d2.balancer.LoadBalancerWithFacilities;
+import com.linkedin.d2.balancer.LoadBalancerWithFacilitiesFactory;
+import com.linkedin.d2.jmx.D2ClientJmxManager;
+import com.linkedin.d2.xds.Node;
+import com.linkedin.d2.xds.XdsChannelFactory;
+import com.linkedin.d2.xds.XdsClient;
+import com.linkedin.d2.xds.XdsClientImpl;
+import com.linkedin.d2.xds.XdsToD2PropertiesAdaptor;
+import com.linkedin.r2.util.NamedThreadFactory;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+
+
+public class XdsLoadBalancerWithFacilitiesFactory implements LoadBalancerWithFacilitiesFactory
+{
+  @Override
+  public LoadBalancerWithFacilities create(D2ClientConfig config)
+  {
+    D2ClientJmxManager d2ClientJmxManager = new D2ClientJmxManager(config.d2JmxManagerPrefix, config.jmxManager);
```

Comment:
Do we need to give xDS data another Jmx prefix? How does it look like in Jmx UI?